scripts/start_civ.sh

```diff
@@ -646,6 +651,10 @@ function launch_guest() {
               $GUEST_EXTRA_QCMD \
     "
 
+    if [[ $PT_ENABLE == "false" ]]; then
+	EXE_CMD+="$USB_STATIC_OPTION \
+	"
+    fi
```

Comment:
remove these, instead, add $GUEST_USB_XHCI_OPT between line644 and line 645

Reply:
ACK